### DISCUSSION TITLE
Is accessing multiple DBs simultaneously with nocodb possible?

### DISCUSSION THREAD

**Discussion by ovizii** [nocodb/nocodb#909] [Features]
Let me explain:

App1 uses DB1 and is my invoicing system. 
App2 uses DB2 and is basically an app that tracks my expenses

Now I thought nocodb would enable me to create a new project in nocodb and pull in data from DB1 AND DB2. 

The manual work flow I am trying to replace is this one:
- create a XLS file every year
- export or copy/paste info from my invoicing app into it
- export or copy/paste info from my expense tracker
- prettify and consolidate the XLS then hand it over to my tax consultant

Is this on the road map? Is nobody interested in anything like this? This seemed to be the most obvious use case to me, aka using nocodb to access and present info from multiple sources (ideally also external sources i.e. imdb.com or similar public sources)

**Comment by ovizii** [comment]
I'm still not sure if this was a dumb question as nobody answered but meanwhile I had a look at Metabase (different purpose) with the same intent and got the reply that: "No, Metabase does not support queries across databases. You would need to setup a warehouse" 

I'm still wondering if nocodb can do this, if it makes no sense? Nobody else wants this? No idea, I would love any feedback. 

Let me try and give some use case: 
if a new invoice is issued in App1 DB1 it should reflect in App2 DB2 and its status changes this should trigger somethign else in App2 DB2. 

Hence my question about accessing multiple DBs with nocodb or is the right way for this to add an automation to the DB which should trigger this?

**Reply by wingkwong** [reply]
We're working on the multi-source feature and should be available in the next feature release.

**Reply by ovizii** [reply]
Awesome! Thanks for the feedback. 
No rush though, I was just confused whether I had asked a silly question :-)

**Reply by glarrain-cdd** [reply]
Hi @wingkwong @ovizii
Any news regarding this feature request? It would be a killer feature.
Thanks

**Reply by overholted** [reply]
I also would like to know what the status of this feature is.
